CHANGELOG.asciidoc

```diff
@@ -28,7 +28,10 @@ This release also includes changes from <<release-3-3-10, 3.3.10>>.
 * Expanded the use of `by(String)` modulator so that it can work on `Map` as well as `Element`.
 * Improved error messaging for `by(String)` so that it is more clear as to what the problem is
 * Bump to Netty 4.1.42
```

Comment:
The last line is the one I want to add in CHANGELOG, but GitHub complains that there is a conflict and I had to add these 3 lines above (GitHub still complains though). I am not clear what should be the right result here.

Please let me know if I need to modify this part.

Reply:
I don't think your PR should contain any lines beyond the last one in this diff. I would just revert your CHANGELOG modification in your branch, rebase the whole PR on tp34 and then add your one line back (and squash) so that it's clean. does that make sense? 

Worst case, if it's taking too much time and you can't figure it out, just revert your CHANGELOG modification, copy/paste your CHANGELOG entry to the comments on this PR and I'll add it manually on merge. thanks.

Reply:
@spmallette Ok, this happened because my `tp34` is behind from `upstream`. I updated `tp34` and rebased this PR onto it, now the conflict is gone. It should be good now.